crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -31,13 +31,22 @@ def initialize(backup)
     end
 
     def upgrade
-      knife_files
-      crowbar_files
+      case ENV["CROWBAR_VERSION"]
+      when "3.0"
```

Comment:
Yeah, true so you should check the backup version here. Otherwise it's totally confusing...